pkg/core/ioutil/editor/linux.go

```diff
@@ -1,4 +1,5 @@
 // nolint
+//go:build !windows
```

Comment:
Does that replace `// +build !windows`?

Reply:
That was added by golang. Maybe I have different version. Should I resolve that?

Reply:
I have no idea, it could be how to do it with new versions or something.